Example/CustomActionControllers/Skype.swift

```diff
@@ -174,9 +179,9 @@ public class SkypeActionController: ActionController<SkypeCell, String, UICollec
                 me.contextView.diff = -pixels
                 me.contextView.setNeedsDisplay()
                 
-                if self?.collectionView.frame.origin.y > self?.view.bounds.size.height {
+                if self!.collectionView.frame.origin.y > self!.view.bounds.size.height {
```

Comment:
please don't use forced unwrapping. Here you can use (we should have used) the local variable `me`